### DISCUSSION TITLE
Some Web radio stations no longer play

### DISCUSSION THREAD

**Discussion by Sagnasty** [rern/rAudio#1382] [Fixed Issues]
Running rAudio 20230630

I noticed today that all of my **streamtheworld** based web radio stations no longer work.

I have been using these web radio stations for quite some time and I believe this problem was recently introduced.

example working web radio link: https://18183.live.streamtheworld.com/WCSXFMAACIHR_SC

This web radio link will play directly in Firefox and VLC

All of my remaining web radio links (including radio paradise links) are working fine.

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

Jim

**Comment by rern** [comment]
Firefox:
GET
`https://18183.live.streamtheworld.com/WCSXFMAACIHR_SC      [HTTP/1 403 Forbidden 1642ms]`

Chrome:
`Failed to load resource: the server responded with a status of 403 ()`

Edge:
![screenshot](https://github.com/rern/rAudio/assets/10094236/a221fc8a-842c-44d3-af54-36f3b3902c03)

**Reply by Sagnasty** [reply]
Confirmed that streamtheworld has changed things a bit as these webradio links also failed with i20230521 (which I know for fact was working before).

Had to use the Firefox web developer tools to see the 403 access issue on the alternative streaming paths as I can still play the stream (there must be an authentication remnant stored on my PC from a previous session).
